src/main/kotlin/org/usecase/locale/LangDefinition.kt

```diff
@@ -0,0 +1,28 @@
+package org.usecase.locale
+
+import org.usecase.exception.response.BadRequestException
+import org.valiktor.ConstraintViolationException
+import java.util.*
+
+open class LangDefinition(bundleName: String) {
+    private val bundle: ResourceBundle = ResourceBundle.getBundle(bundleName)
+
+    operator fun get(o: String): String {
+        return try {
+            bundle.getString(o)
```

Comment:
We should use String.format here, even without a variable, so we make the use of `%` more concise.

And we also need another method to pass some variables to the String.format